project/SparkBuild.scala

```diff
@@ -671,9 +670,9 @@ object KubernetesIntegrationTests {
     (Test / javaOptions) ++= Seq(
       s"-Dspark.kubernetes.test.deployMode=${deployMode.getOrElse("minikube")}",
       s"-Dspark.kubernetes.test.imageTag=${imageTag.value}",
-      s"-Dspark.kubernetes.test.namespace=${namespace.value}",
       s"-Dspark.kubernetes.test.unpackSparkDir=$sparkHome"
     ),
+    (Test / javaOptions) ++= namespace.map("-Dspark.kubernetes.test.namespace=" + _),
```

Comment:
Shouldn't this be: `(Test / javaOptions) += namespace.map("-Dspark.kubernetes.test.namespace=" + _).getOrElse("default"),` ?
The old `default` value from line 637 seems to be lost now.

Reply:
This change was intentional, @martin-g .
- This new default is consistent with `Maven` integration test's default test coverage which uses random namespaces.
- Last time, we missed the namespace propagation issue because SBT only tested with a single `default` namespace at that time. I fixed that later via ed3ea989f97634374789d173f1cc932230bf3aa1 .

After this change, we have the same test coverage and behavior in Maven/SBT.

Reply:
Thanks for explaining, @dongjoon-hyun !